src/scratchtocatrobat/converter/converter.py

```diff
@@ -1598,8 +1614,22 @@ def _convert_create_clone_of_block(self):
         if len(base_sprite) == 0:
             return catbricks.NoteBrick("Can't convert Clone-Brick with no argument.")
 
+        if base_sprite == "_myself_":
```

Comment:
Please replace:
`if base_sprite == "_myself_":`
with:
`if base_sprite == "_myself_" or base_sprite == self.sprite.getName():`
since the currently converted sprite will be added to the sprite list after all blocks have been converted.

So, in case the "createCloneOf" block is set to "NAME_OF_THIS_SPRITE" (which is the same as "_myself_") the converter would accidentally create a second Sprite object with the same name and add it to the clone_sprites list of the Context object.